frontend/www/js/omegaup/components/problem/creator/cases/AddPanel.test.ts

```diff
@@ -74,11 +78,111 @@ describe('AddPanel.vue', () => {
     const groups = store.state.casesStore.groups;
 
     expect(groups.length).toBe(1);
-    expect(groups[0].name).toBe('groupname');
+    expect(groups[0].name).toBe(testGroup);
     expect(groups[0].autoPoints).toBe(false);
     expect(groups[0].points).toBe(10);
     expect(groups[0].ungroupedCase).toBe(false);
+
+    const groupId = groups?.[0].groupID;
+
+    wrapper.setData({ tab: 'case' });
+    await Vue.nextTick();
+
+    const caseNameInput = wrapper.find('input[name="case-name"]');
+    const casePointsInput = wrapper.find('input[name="case-points"]');
+    const caseGroupName = wrapper.find('select[name="case-group"]');
+
+    await caseNameInput.setValue('groupedtestcase');
+    await casePointsInput.setValue(10);
+    await caseGroupName.setValue(groupId);
+
+    await wrapper.find('form').trigger('submit.prevent');
+
+    const groupedCases = store.state.casesStore.groups?.[0].cases;
+    expect(groupedCases.length).toBe(1);
+    expect(groupedCases[0].name).toBe('groupedtestcase');
+    expect(groupedCases[0].groupID).toBe(groupId);
+    expect(groupedCases[0].points).toBe(10);
```

Comment:
Now you can access to `groupedCase` this way:
```suggestion
    expect(groupedCase.name).toBe('groupedtestcase');
    expect(groupedCase.groupID).toBe(groupId);
    expect(groupedCase.points).toBe(10);
```

Reply:
Thanks!